index.html

```diff
@@ -280,7 +281,7 @@ <h2 class="post-albums__scholar--name" itemprop="name">Kirssy Fernández Roja </
 			<h3 class="page-footer--lab" itemprop="legalname">Complex Systems Optimization Lab </h3>
 			<h2 class="page-footer--name" itemprop="alternateName">COSOLA</h2>
 			<h4 class="page-footer--academy" itemprop="brand">MACILE Education Program</h4>
-			<p class="page-footer--text"> Copyright 2007-2019 by COSOLA. All rights reserved</p>
+			<p class="page-footer--text"> Copyright 2007-2020 by COSOLA. All rights reserved</p>
```

Comment:
To avoid having to manually update the copyright end year every time, we should just use a javascript function. I don't want to hold up this story for that, so I'll add it to the backlog #70